src/app/clusters/energy-evse-server/EnergyEvseTestEventTriggerHandler.h

```diff
@@ -57,6 +57,8 @@ enum class EnergyEvseTrigger : uint64_t
     kEVChargeDemand = 0x0099000000000004,
     // EV Charge Demand Test Event Clear | Simulate the EV becoming fully charged
     kEVChargeDemandClear = 0x0099000000000005,
+    // EV Charge TimeOfUse Mode | Simulate putting the EVSE into a Mode with the TimeOfUse tag included
+    kEVTimeOfUseMode = 0x0099000000000006,
```

Comment:
As above perhaps we need a TimeOfUseMode clear trigger which is sent at the end of the test.

Reply:
Added but will be fully implemented when issue #34249 is addressed.